crates/goose/src/message.rs

```diff
@@ -156,6 +158,37 @@ impl From<Content> for MessageContent {
     }
 }
 
+impl From<PromptMessage> for Message {
+    fn from(prompt_message: PromptMessage) -> Self {
+        // Create a new message with the appropriate role
+        let message = match prompt_message.role {
+            PromptMessageRole::User => Message::user(),
+            PromptMessageRole::Assistant => Message::assistant(),
```

Comment:
the code herer looks fine.

in the CLI and server:
if we invoked a PromptMessage that starts with assistant role - i _think_ that would error because the CLI assumes its user msg and we also require alternative between user & assistant. can you check this?